docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml

```diff
@@ -54,6 +55,17 @@ ResumptionDataProcessor -> ResumptionDataProcessor : prevent RAI response until
 RAI2 -> ResumptionDataProcessor : StartResumption
 ResumptionDataProcessor -> ExtensionPendingResumptionHandler : Restore subscriptions data : **sub2** for app2
 ExtensionPendingResumptionHandler -> ExtensionPendingResumptionHandler : Create REQUEST2  **sub2**. \n Freeze REQUEST2 until response to REQUEST1  will not be received
```

Comment:
@LuxoftAKutsan Probably I didn't make my question clear during the meeting.
my question is assuming that app2 need both data of sub_2 an sub_3, 
will this step generate one frozen REQUEST2  with both data items, or will this step generate two separate  frozen  requests (or one frozen request for sub_2 plus on sent request for sub_3), each of them is corresponding to sub_2 and sub_3?

Reply:
@yang1070 according to our implementation, even if app2 need both `sub_2`,`sub_3` ... `sub_N` but another subscription request is still in progress (response not received yet) then SDL will create one frozen REQUEST2 with all the data items required for app2 and send nothing to HMI on this step.

Reply:
sounds good. thanks